official/resnet/keras/keras_common.py

```diff
@@ -358,6 +362,15 @@ def data_prefetch_with_slack():
     _monkey_patch_org_create_device_dataset()
 
 
+def set_cudnn_batchnorm_mode():
+  """Set CuDNN batchnorm mode for better performance. Note that the spatial
+     persistent mode may lead to accuracy losses for certain models."""
+  if FLAGS.batchnorm_spatial_persistent:
+    os.environ['TF_USE_CUDNN_BATCHNORM_SPATIAL_PERSISTENT'] = '1'
+  else:
+    os.environ.pop('TF_USE_CUDNN_BATCHNORM_SPATIAL_PERSISTENT', None)
```

Comment:
My understanding is that this is the best way to remove an ENV VAR.

`del os.environ['MYVAR']`

Reply:
Got it will make the change.